pkgs/development/tools/misc/hydra/unstable.nix

```diff
@@ -113,57 +115,78 @@ let
         TermSizeAny
         TermReadKey
         Test2Harness
-        TestPostgreSQL
+        TestMore
+        (buildPerlModule {
```

Comment:
Why not add this to perlPackages?

Reply:
This might be a version specifically needed by hydra? I'm not sure, maybe @grahamc can explain.

Reply:
I looked into this and it should be possible to use version 1.29 of the library: https://github.com/TJC/Test-postgresql/issues/40